data/scenarios/README.md

```diff
@@ -260,7 +260,7 @@ table.
 | `name`        |          | `string`              | The name of the robot.  This shows up in the list of robots in the game (F2), and is also how the robot will be referred to in the [world](#world) `palette`.                                                                                                                                                                                                                                                                  |
 | `description` | `[]`     | `string list`         | A description of the robot, given as a list of paragraphs.  This is currently not used for much (perhaps not at all?).                                                                                                                                                                                                                                                                                                         |
 | `loc`         | `null`   | `int × int`           | An optional (x,y) starting location for the robot.  If the `loc` field is specified, then a concrete robot will be created at the given location.  If this field is omitted, then this robot record exists only as a *template* which can be referenced from a [cell](#cells) in the [world](#world) `palette`.  Concrete robots will then be created wherever the corresponding palette character is used in the world `map`. |
-| `dir`         |          | `int × int`           | The starting orientation of the robot, expressed as a vector.  Every time the robot executes a `move` command, this vector will be added to its position.  Typically, this is a unit vector in one of the four cardinal directions, although there is no particular reason that it has to be.                                                                                                                                  |
+| `dir`         |          | `int × int`           | An optional starting orientation of the robot, expressed as a vector.  Every time the robot executes a `move` command, this vector will be added to its position.  Typically, this is a unit vector in one of the four cardinal directions, although there is no particular reason that it has to be. When ommited, the robots direction will be the zero vector.                                                                                                 |
```

Comment:
We should also add `[0,0]` to the second column of the table, which is used to indicate default values for optional fields.
```suggestion
| `dir`         | `[0,0]`  | `int × int`           | An optional starting orientation of the robot, expressed as a vector.  Every time the robot executes a `move` command, this vector will be added to its position.  Typically, this is a unit vector in one of the four cardinal directions, although there is no particular reason that it has to be. When omitted, the robot's direction will be the zero vector.                                                                                                 |
```